### ISSUE TITLE
Component doesn't icon mdi:weather-partly-cloudy

### ISSUE THREAD

**Issue by Spirituss** [sgttrs/lovelace-weather-card-chart#28]
**Home Assistant release with the issue:**

0.98

**Last working Home Assistant release (if known):**
0.97.4

**Operating environment (Hass.io/Docker/Windows/etc.):**

Hass.io / Docker / Raspbian Buster

Some earlier working icons stop working in new HA version 0.98. For example, the cloudy weather icon now are empty, while lovelace shows it correctly in both glance and entities. 
This is an example of problem icon: mdi:weather-partly-cloudy.
See the screenshots below:
 
Card:
<img width="441" alt="Screen Shot 2019-09-03 at 21 41 08" src="https://user-images.githubusercontent.com/47206760/64199884-040db680-ce94-11e9-9e4b-a2cc52237b69.png">

Opened card:
<img width="55" alt="Screen Shot 2019-09-03 at 21 41 28" src="https://user-images.githubusercontent.com/47206760/64199886-040db680-ce94-11e9-8802-503984edf661.png">

**Comment by sgttrs** [comment]
Check the new version of the file from the repository, please.

**Comment by papperone** [comment]
Why you are not using HACS in order to trigger automatic notification of new version of your card?

**Comment by Spirituss** [comment]
@sgttrs What repository should I check? I have all the latest version components.
@papperone I do use HACS for custom-weather-card too:
<img width="519" alt="Screen Shot 2019-09-19 at 13 43 25" src="https://user-images.githubusercontent.com/47206760/65237620-a9a36580-dae3-11e9-88d8-ebf72e36b8af.png">

Still the same spaces instead of icons:
<img width="428" alt="Screen Shot 2019-09-19 at 13 45 57" src="https://user-images.githubusercontent.com/47206760/65237761-e8392000-dae3-11e9-93f2-d8811941507a.png">

**Comment by papperone** [comment]
I manage to add it in HACS but as the github does not update version number no new updates are triggered; anyhow I removed completely the card, reinstall the supposed-to-be fixed but same problem, some icons are not shown (for sure the "weather-partly-cloudy" one, no idea if others as well)

PS: I would like to add support for ITALIAN language, but I don't know how to submit a request here in GitHub, how can I help here?

**Comment by Spirituss** [comment]
Still waiting for the fix. 0.99.3

**Comment by papperone** [comment]
same here, even upgrading the cards manually from latest version in Github does not restore the missin icons :(

**Comment by sgttrs** [comment]
@papperone Translate the values below, I will add it in the next release.
```
Temperature: 
Temperature night: 
Precipitations:
hPa:
m/s:
mm:
Directions: 'N', 'N-NE', 'NE', 'E-NE', 'E', 'E-SE', 'SE', 'S-SE', 'S', 'S-SW', 'SW', 'W-SW', 'W', 'W-NW', 'NW', 'N-NW', 'N'
```

**Comment by sgttrs** [comment]
Please, check v0.2.6.

**Comment by papperone** [comment]
just upgraded to 0.2.6, even reboot Hassio completely but still icons are missing :(

-------------------
here's the translation for Italian:

Temperatura: 
Temperatura notturna: 
Precipitazioni:
hPa:
m/s:
mm:
Directions: 'N', 'N-NE', 'NE', 'E-NE', 'E', 'E-SE', 'SE', 'S-SE', 'S', 'S-SO', 'SO', 'O-SO', 'O', 'O-NO', 'NO', 'N-NO', 'N'

**Comment by sgttrs** [comment]
@papperone Did you clear your browser cache after updating?

**Comment by papperone** [comment]
I did, but earlier today I tried form my phone and it works now, with the icons back!!
I'll check again from the PC browser after I clean again the cache....

**Comment by Spirituss** [comment]
Yesterday it occasionally started working for me too.

**Comment by Spirituss** [comment]
@papperone Please, carefully test it, I'm going to close the issue.

**Comment by Spirituss** [comment]
Solved in 0.100
